Library/ViewModels/DashboardReferrerRowStackViewViewModel.swift

```diff
@@ -38,15 +38,15 @@ public final class DashboardReferrerRowStackViewViewModel: DashboardReferrerRowS
 
     self.pledgedText = countryReferrer
       .map { country, referrer in
-        Format.currency(referrer.pledged, country: country) + " ("
+        Format.currency(Int(referrer.pledged), country: country) + " ("
```

Comment:
So we're losing precision here again, no? Is the change just to keep precision through to Koala tracking?